packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -474,47 +474,56 @@ describe('plugin-conversation', function () {
         webex3.internal.conversation.listActivities({conversationUrl: convo1.url})
           .then((convoActivities) => {
             const activityURLs = [];
+            const expectedActivities = [];
 
-            convoActivities.map((a) => {
+            convoActivities.forEach((a) => {
               if (a.verb === 'post') {
                 activityURLs.push(a.url);
+                expectedActivities.push(a);
               }
-
-              return activityURLs;
             });
-            webex3.internal.conversation.bulkActivitiesFetch(activityURLs)
+
+            return webex3.internal.conversation.bulkActivitiesFetch(activityURLs)
               .then((bulkFetchedActivities) => {
-                assert.equal(bulkFetchedActivities, convoActivities);
+                assert.lengthOf(bulkFetchedActivities, expectedActivities.length);
+                assert.equal(bulkFetchedActivities[0].object.displayName, expectedActivities[0].object.displayName);
+                assert.equal(bulkFetchedActivities[1].object.displayName, expectedActivities[1].object.displayName);
               });
           }));
 
-      it('retrieves activities from multiple conversations', () =>
-        webex3.internal.conversation.listActivities({conversationUrl: convo1.url})
-          .then((convo1Activities) => {
-            const activityURLs = [];
+      it('retrieves activities from multiple conversations', () => {
```

Comment:
these tests were not working at all, but it was not catching the errors because it was not returning the promise. fixed the tests